tests/analytics_test.py

```diff
@@ -97,6 +96,10 @@ def test_shodan_query_with_various_limits(self, limit, expected_count, mock_shod
         mock_shodan.return_value = mock_shodan_api
 
         os.environ["YETI_SHODAN_API_KEY"] = "test_api_key"
+        if limit:
+            os.environ["YETI_SHODAN_RESULT_LIMIT"] = str(limit)
+        else:
+            os.environ["YETI_SHODAN_RESULT_LIMIT"] = ""
```

Comment:
Do we want this or to `del` the env variable?

Reply:
fixed in 48a6ce0820487573af5c33c7185482433c533901